packages/instantsearch.js/CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [4.65.0](https://github.com/algolia/instantsearch/compare/instantsearch.js@4.64.3...instantsearch.js@4.65.0) (2024-02-13)
+
+
+### Features
+
+* **instantsearch-ui-components:** update build ([#6037](https://github.com/algolia/instantsearch/issues/6037)) ([b4ed187](https://github.com/algolia/instantsearch/commit/b4ed187d5ffab1067b68f553278e0d5feffd3527))
+* **instantsearch.js:** replace `Highlight` and `cx` ([#6030](https://github.com/algolia/instantsearch/issues/6030)) ([ff8b345](https://github.com/algolia/instantsearch/commit/ff8b3458f147349f4c041ded7a1b426b45b28b5a))
+
+
```

Comment:
### Bug Fixes

* **autocomplete:** add indexId to type ([#6050](https://github.com/algolia/instantsearch/issues/6050)) ([5f60268](https://github.com/algolia/instantsearch/commit/5f60268b1d74b33e93c17d1c9677e8abce7f6429))